plugins/module_utils/foreman_helper.py

```diff
@@ -245,7 +245,7 @@ def _generate_resource_payload(self, action_params, data):
         resource_payload = {}
 
         for param in action_params:
-            if param.expected_type == 'hash':
+            if param.expected_type == 'hash' and param.params:
```

Comment:
That's interesting. Are there really apidocs that have `expected_type=hash` but don't provide nested params?

Reply:
(asking, because I've recently ported that function to Apypie directly, and thus would have to also port that fix)

Reply:
Yes, there was one.
`cat default.json | jq '.docs.resources.compute_attributes.methods[0].params[]|select(.name=="compute_attribute").params'`

Reply:
found it! https://github.com/Apipie/apypie/pull/34